src/GeneralStateTestsFiller/stPreCompiledContracts2/CALLCODEBlake2fFiller.json

```diff
@@ -0,0 +1,217 @@
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0xb63a380cb2897d521994a85234ee2c181b5f844d2c624c002677e9703449d2fb",
+              "0x03": "0xa551b3a8333bcdf5f2f7e08993d53923de3d64fcc68c034e717b9293fed7a421"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999916431"
+          }
+        }
+      },
+      {
+        "indexes": {
+          "data": [8],
+          "gas": -1,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0x08c9bcf367e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5",
+              "0x03": "0xd182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999999917272"
+          }
+        }
+      },
+      {
+        "indexes": {
+          "data": [9],
+          "gas": 0,
+          "value": -1
+        },
+        "network": [">=Istanbul"],
+        "result": {
+          "1000000000000000000000000000000000000000": {
+            "storage": {
+              "0x01": "0x01",
+              "0x02": "0x6d2ce9e534d50e18ff866ae92d70cceba79bbcd14c63819fe48752c8aca87a4b",
+              "0x03": "0xb7dcc230d22a4047f0486cfcfb50a17b24b2899eb8fca370f22240adb5170189"
+            }
+          },
+          "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+            "balance": "999999999999991916420"
+          }
+        }
+      }
+    ],
+    "pre": {
+      "1000000000000000000000000000000000000000": {
+        "balance": "",
+        "code": "{ (CALLDATACOPY 0 0 (CALLDATASIZE)) [[1]] (CALLCODE (GAS) 9 0 0 (CALLDATASIZE) 1000 64) [[2]](MLOAD 1000) [[3]](MLOAD 1032)}",
+        "nonce": "0",
+        "storage": {}
+      },
+      "a94f5374fce5edbc8e2a8697c15331677e6ebf0b": {
+        "balance": "1000000000000000000000",
+        "code": "",
+        "nonce": "0",
+        "storage": {}
+      }
+    },
+    "transaction": {
+      "// data": [
+        "Input is Packed as per EIP-152 requirements",
+        "Output is [0] - result - [1] high 256 bits [2] low 256 bits",
+        "Test cases 0 through 7 are from the EIP, case 8 from the EIP is in stTimeConsuming",
+        "0 - no bytes",
+        "1 - too few bytes",
+        "2 - too many bytes",
+        "3 - f flag wrong value",
+        "4 - RFC 7693 Appendix-A, zero rounds, final",
+        "5 - RFC 7693 Appendix-A, twelve rounds, final",
+        "6 - RFC 7693 Appendix-A, twelve rounds, not final",
+        "7 - RFC 7693 Appendix-A, one round, final",
+        "8 - all zero input, correct length, zero rounds, not final",
+        "9 - RFC 7693 Appendix-A, 8 million rounds"
+      ],
+      "data": [
+        "0x",
```

Comment:
Another test suggestion: non-empty data, but fewer than 4 bytes, to catch possible errors of trying to calculate the precompile cost out of it

Reply:
added as data 10